File: informatykaEuropejczyka/Rozdzial2/prog2_33.cpp
//sortowanie tekstu przez wybor
#include <iostream>
#include <cstring>
using namespace std;

string sortuj (string s)
{
 int dl=s.size(), k;
 char pom;
 for (int i=0;i<dl-1;i++)
 {
  k=i;
  for (int j=i+1;j<dl;j++)
   if (s[j]<s[k]) k=j;
  pom=s[k];
  s[k]=s[i];
  s[i]=pom;
 }
 return s;
}

int main()
{
 string s;
 cout<<"podaj tekst bez spacji:"<<endl;
 cin>>s;
 cout<<"posortowany tekst:\n"<<sortuj(s)<<endl;
 return 0;
}

File: informatykaEuropejczyka/Rozdzial2/prog2_33.py
#sortowanie tekstu przez wybor

def sortuj (s):
    dl=len(s)
    s=list(s)
    for i in range(dl-1):
        k=i
        for j in range(i+1,dl):
            if s[j]<s[k]:
                k=j
        s[k],s[i]=s[i],s[k]
    s=''.join(s)
    return s

#print(sortuj("zxcvbnmas"))

